OrbitGl/CallStackDataView.cpp

```diff
@@ -192,15 +194,9 @@ CallStackDataView::CallStackDataViewFrame CallStackDataView::GetFrameFromIndex(
   }
 
   uint64_t address = callstack_.GetFrame(index_in_callstack);
-  FunctionInfo* function = nullptr;
-  std::shared_ptr<Module> module = nullptr;
-
-  const std::shared_ptr<Process> process = GOrbitApp->GetCaptureData().process();
-  CHECK(process != nullptr);
-  // TODO(kuebler): Get rid of locks on caller side
-  ScopeLock lock(process->GetDataMutex());
-  function = process->GetFunctionFromAddress(address, false);
-  module = process->GetModuleFromAddress(address);
+
+  const FunctionInfo* function = GOrbitApp->GetCaptureData().FindFunctionByAddress(address, false);
+  ModuleData* module = GOrbitApp->GetCaptureData().FindModuleByAddress(address);
```

Comment:
can this also be const?

Reply:
Nope, since this pointer will potentially be used for symbol loading